src/EmergencyContent.elm

```diff
@@ -46,26 +47,26 @@ renderEmergencyPanel viewportWidth =
                     , text (t EmergencyDomesticAbuseInfo)
                     ]
                 , li [ css [ marginBottom (rem 1) ] ]
-                    [ a [ href (t WomensAidHref) ]
+                    [ a [ css [ linkStyle ], href (t WomensAidHref) ]
                         [ text
                             (t EmergencyWomensAidLink)
                         ]
                     , text " - "
                     , text (t EmergencyWomensAidInfo)
                     ]
                 ]
-            , p [] [ a [ href (t SeaOrganisationsResourceHref) ] [ text (t EmergencyOtherOrganisationsLink) ] ]
+            , p [] [ a [ css [ linkStyle ], href (t SeaOrganisationsResourceHref) ] [ text (t EmergencyOtherOrganisationsLink) ] ]
             ]
         ]
 
 
 renderPhoneNumber : Float -> String -> Html msg
 renderPhoneNumber viewportWidth phonenumber =
     if viewportWidth <= Theme.maxMobile then
-        a [ href ("tel:" ++ String.replace " " "" phonenumber) ] [ text phonenumber ]
+        a [ css [ linkStyle ], href ("tel:" ++ String.replace " " "" phonenumber) ] [ text phonenumber ]
 
     else
-        text phonenumber
+        b [] [ text phonenumber ]
```

Comment:
b elements don't explicitly set styling anymore, but I don't know which browser would not make this bold.

Reply:
OK so you think I need to set font-weight bold or can ignore?

Reply:
It's probably fine but if you want to be sure you could
```
        , typeSelector "b"
            [ fontWeight (int 700)
            ]
```
in Theme.elm > globalStyles